### ISSUE TITLE
How to delete all posts/comments by a user?

### ISSUE THREAD

**Issue by blez4o** [e107inc/e107#3780]

**Comment by tgtje** [comment]
System has prune, but posts etc stay without user name.
This is a general question or maybe you post it as enhancement ?
Anyway, imo  best option is by database (sql) manual based = use tables and do colum sorting example :

**Comment by Jimmi08** [comment]
@tgtje If you ban a user, their posts are still displayed on forum.  

There is an option now because GDPR to remove all data (not tested how it works, if it deletes everything or anonymized), so you can log from admin area as that user and try it.  Theoretically, not tested.

**Comment by tgtje** [comment]
Exactly, hence dbase method. As for GDPR, that should look into it, but not every post etc. (ex. general post or comment) could be under privacy.. Is a tricky interpretation in such cases.
It would also be if a user IS removed or does it by him/her self.. etc... long discussion

**Comment by Jimmi08** [comment]
@tgtje I know this too well.  But I think he is talking about spam user and it was really annoying to delete everything manually.

**Comment by Jimmi08** [comment]
Go to e107_admin/comment.php file and add the option to filter by user
'filter' => 'true',	

![image](https://user-images.githubusercontent.com/5429548/97552614-c2fc2e00-19d4-11eb-9421-a96cb2bb424e.png)

Then just select your user in filter, mark all comments and use batch delete option 

![image](https://user-images.githubusercontent.com/5429548/97552832-06ef3300-19d5-11eb-9bcb-9e0892618347.png)

![image](https://user-images.githubusercontent.com/5429548/97552890-14a4b880-19d5-11eb-97db-caacbf901d6d.png)

![image](https://user-images.githubusercontent.com/5429548/97552935-20907a80-19d5-11eb-8352-65bee818be04.png)
